pkg/apis/kubermatic/v1/etcd_restore.go

```diff
@@ -36,9 +36,12 @@ const (
 
 	// EtcdRestorePhaseCompleted value indicating that the old Etcd statefulset has completed successfully.
 	EtcdRestorePhaseCompleted EtcdRestorePhase = "Completed"
+
+	// EtcdRestorePhaseEtcdLauncherNotEnabled value indicating that etcd-launcher is not enabled.
+	EtcdRestorePhaseEtcdLauncherNotEnabled EtcdRestorePhase = "EtcdRestorePhaseEtcdLauncherNotEnabled"
```

Comment:
Sorry I should have caught this earlier, but this should remove the `EtcdRestorePhase` prefix. It's not on the other phases either.

```suggestion
	EtcdRestorePhaseEtcdLauncherNotEnabled EtcdRestorePhase = "EtcdLauncherNotEnabled"
```

Same goes for the enum value of course.